lib/OOCEapps/PkgUpd/base.pm

```diff
@@ -1,6 +1,12 @@
 package OOCEapps::PkgUpd::base;
 use Mojo::Base -base;
 
+# packages where odd minor versions mean development version
+my @oddities = (
+    'system/library/dbus',
+    'library/glib2',
+);
+
```

Comment:
even with this name it feels odd to have package specific information at this level

Reply:
The other option is to make a new module that handles these packages.
It would just be a sub-class of base with the extra grep in it.

Reply:
well I guess @hadfl will have an opinion as he designed it :)